<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serial</class>
 <widget class="QMainWindow" name="serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serial</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,5">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>端口名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxPortName"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>比特率：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBaud">
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>数据位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxData">
            <item>
             <property name="text">
              <string>Data5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data8</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>停止位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxStop">
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>校验位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxParity">
            <item>
             <property name="text">
              <string>奇校验</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>偶校验</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>space</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>流控制：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxFlow">
            <item>
             <property name="text">
              <string>硬件控制</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>软件控制</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonOpen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>打开串口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonClose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>关闭串口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="pushButtonSend">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonClear">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
